src/Core/Components/List/FluentAutocomplete.razor

```diff
@@ -196,15 +197,25 @@
 
     private RenderFragment RenderSelectedOptions => __builder =>
     {
-        if (SelectedOptions != null)
+        var selectedOptions = new List<TOption>();
+        if(Multiple && SelectedOptions.Any())
```

Comment:
`SelectedOptions` is nullable. You need to test it.
We prefer to use the `if` syntax using "curly braces" (one on each line).

Reply:
Fixed